Content.Shared/Chasm/ChasmSystem.cs

```diff
@@ -51,18 +52,33 @@ private void OnStepTriggered(EntityUid uid, ChasmComponent component, ref StepTr
         if (HasComp<ChasmFallingComponent>(args.Tripper))
             return;
 
-        var falling = AddComp<ChasmFallingComponent>(args.Tripper);
+        StartFalling(uid, component, args.Tripper);
+
+        // checking for strap is necessary so that buckled entities actually have the visuals and can't just
+        // unbuckle to prevent death
+        if (!TryComp<StrapComponent>(uid, out var strap))
+            return;
+
+        foreach (var ent in strap.BuckledEntities)
+        {
+            // don't play sound for any buckled ents since we'll have already played it once
+            StartFalling(uid, component, ent, playSound: false);
+        }
+    }
```

Comment:
Doesn't this also affect stuff like entitystorage? Seems better to just handle new collisions rather than adding special-cases for every scenario.

Reply:
ehh yeah im just gonna remove the special casing

although this is only really relevant for entity children that should be visually falling, so this doesnt affect entitystorage/containers which occlude